support/Pipelines/Bbh/FindHorizon.py

```diff
@@ -0,0 +1,231 @@
+    """Volume data representation of a Strahlkorper, can be written to H5."""
+    coords = cartesian_coords(strahlkorper)
+    return [
+        spectre_h5.ElementVolumeData(
+            element_name="Horizon",
+            components=[
+                spectre_h5.TensorComponent(
+                    "InertialCoordinates_" + "xyz"[d], coords[d]
+                )
+                for d in range(3)
+            ],
+            extents=strahlkorper.physical_extents,
+            basis=2 * [Basis.SphericalHarmonic],
+            quadrature=[
+                Quadrature.Gauss,
+                Quadrature.Equiangular,
+            ],
+        )
+    ]
+
+
+def find_horizon(
+    h5_files: Union[str, Sequence[str]],
+    subfile_name: str,
+    obs_id: int,
+    obs_time: float,
+    l_max: int,
+    initial_radius: float,
+    center: Sequence[float],
+    output: Optional[str] = None,
+    output_coeffs_subfile: Optional[str] = None,
+    output_coords_subfile: Optional[str] = None,
+    tensor_names: Optional[Sequence[str]] = None,
+):
+    """Find an apparent horizon in volume data.
+
+    Arguments:
+      h5_files: List of H5 files containing volume data or glob pattern.
+      subfile_name: Name of the volume data subfile in the 'h5_files'.
+      obs_id: Observation ID in the volume data.
+      obs_time: Time of the observation.
+      l_max: Maximum l-mode for the horizon search.
+      initial_radius: Initial coordinate radius of the horizon.
+      center: Coordinate center of the horizon.
+      output: Optional. H5 output file where the horizon Ylm coefficients will
+        be written. Can be a new or existing file. Requires either
+        'output_coeffs_subfile' or 'output_coords_subfile' is also specified,
+        or both.
+      output_coeffs_subfile: Optional. Name of the subfile in the H5 output file
+        where the horizon Ylm coefficients will be written.
+        These can be used to reconstruct the horizon, e.g. to initialize
+        excisions in domains.
+      output_coords_subfile: Optional. Name of the subfile in the H5 output file
+        where the horizon coordinates will be written.
+        These can be used for visualization.
+      tensor_names: Optional. List of tensor names in the volume data that
+        represent the inverse spatial metric, extrinsic curvature, and spatial
+        Christoffel symbols, in this order. Defaults to ["InverseSpatialMetric",
+        "ExtrinsicCurvature", "SpatialChristoffelSecondKind"].
+
+    Returns: The Strahlkorper representing the horizon.
+    """
+    if not tensor_names:
+        tensor_names = [
+            "InverseSpatialMetric",
+            "ExtrinsicCurvature",
+            "SpatialChristoffelSecondKind",
+        ]
+    fast_flow = FastFlow(
+        FlowType.Fast,
+        alpha=1.0,
+        beta=0.5,
+        abs_tol=1e-12,
+        truncation_tol=0.01,
+        divergence_tol=1.2,
+        divergence_iter=5,
+        max_its=100,
```

Comment:
Should we allow any of these to be specifiable? These work for relatively spherical horizons, but not sure how they'll do for more distorted ones

Reply:
Yea we can expose any or all of these, I just didn't know which were necessary. We can expose them when we need them, unless you already know which ones are necessary.